tests/OSPSuite.Presentation.Tests/Services/DataImporterSpecs.cs

```diff
@@ -0,0 +1,291 @@
+               new Column() {Name = "Time", Dimension = _dimensionFactory.Dimension("Time"), Unit = new UnitDescription("h")}),
+            new MappingDataFormatParameter("conc  [mg/l]",
+               new Column()
+               {
+                  Name = "Concentration", Dimension = _massConcentrationDimension, Unit = new UnitDescription("mg/l")
+               }),
+            new MappingDataFormatParameter("SD [mg/l]",
+               new Column()
+               {
+                  Name = "Error", ErrorStdDev = "Arithmetic Standard Deviation", Dimension = _massConcentrationDimension,
+                  Unit = new UnitDescription("mg/l")
+               }),
+            new MetaDataFormatParameter("VenousBlood", "Organ", false),
+            new MetaDataFormatParameter(null, "Molecule", false)
+         };
+         _importerConfiguration.CloneParametersFrom(parameterList);
+      }
+
+      [Observation]
+      public void should_import_simple_data_from_excel()
+      {
+         sut.ImportFromConfiguration(_importerConfiguration, _metaDataCategories, _columnInfos, _dataImporterSettings,
+            getFileFullName(
+               "IntegrationSample1.xlsx")).Count.ShouldBeEqualTo(1);
+      }
+
+      [Observation]
+      public void should_import_simple_data_from_csv()
+      {
+         sut.ImportFromConfiguration(_importerConfiguration, _metaDataCategories, _columnInfos, _dataImporterSettings,
+            getFileFullName(
+               "IntegrationSample1.csv")).Count.ShouldBeEqualTo(1);
+      }
+
+      [Observation]
+      public void should_return_empty_on_invalid_file_name()
+      {
+         sut.ImportFromConfiguration(_importerConfiguration, _metaDataCategories, _columnInfos, _dataImporterSettings,
+            getFileFullName(
+               "")).Count.ShouldBeEqualTo(0);
+      }
+
+      [Observation]
+      public void should_return_empty_on_non_existent_file_name()
+      {
+         sut.ImportFromConfiguration(_importerConfiguration, _metaDataCategories, _columnInfos, _dataImporterSettings,
+            getFileFullName(
+               "sample_non_existent.xlsx")).Count.ShouldBeEqualTo(0);
+      }
+
+      [Observation]
+      public void should_throw_on_invalid_file_type()
+      {
+         sut.ImportFromConfiguration(_importerConfiguration, _metaDataCategories, _columnInfos, _dataImporterSettings,
+            getFileFullName(
+               "simple.pkml")).Count.ShouldBeEqualTo(0);
+         A.CallTo(() => _dialogCreator.MessageBoxError(Error.UnsupportedFileType)).MustHaveHappened();
+      }
+
+      [Observation]
+      public void should_return_empty_on_invalid_file_format()
+      {
+         sut.ImportFromConfiguration(_importerConfiguration, _metaDataCategories, _columnInfos, _dataImporterSettings,
+            getFileFullName(
+               "sample1.xlsx")).Count.ShouldBeEqualTo(0);
+      }
+
+      //currently not working
+      /*
+            [Observation]
```

Comment:
should we uncomment the test and just add `[Ignore]` ?

Reply:
I have ignored the test for now, but good that you saw that Yuri. It addresses an exception handling issue that we should talk about. I am going to create an issue about this.